net/udp/udp_callback.c

```diff
@@ -83,6 +83,14 @@ static uint16_t udp_datahandler(FAR struct net_driver_s *dev,
   FAR void  *src_addr;
   uint8_t src_addr_size;
 
+#if CONFIG_NET_RECV_BUFSIZE > 0
+  while (iob_get_queue_size(&conn->readahead) > conn->rcvbufs)
+    {
+      iob = iob_remove_queue(&conn->readahead);
+      iob_free_chain(iob, IOBUSER_NET_UDP_READAHEAD);
+    }
+#endif
```

Comment:
does this mean to drop older packets first? why?

Reply:
The behavior here is the result of my test on Linux. If the UDP network storm comes and exceeds the limit of recv, the old packets will be discarded

Reply:
do you mean you followed what linux does?

Reply:
Yes, I can't find a suitable reference except Linux. Do you have any suggestions?

Reply:
no. i just wanted to know why you did this way.